build-system/tasks/runtime-test/index.js

```diff
@@ -700,32 +56,4 @@ module.exports = {
 
 /* eslint "google-camelcase/google-camelcase": 0 */
 
-test.description = 'Runs tests';
-test.flags = {
-  'verbose': '  With logging enabled',
-  'testnames': '  Lists the name of each test being run',
-  'watch': '  Watches for changes in files, runs corresponding test(s)',
-  'saucelabs': '  Runs integration tests on saucelabs (requires setup)',
-  'saucelabs_lite':
-    '  Runs tests on a subset of saucelabs browsers (requires setup)',
-  'safari': '  Runs tests on Safari',
-  'firefox': '  Runs tests on Firefox',
-  'edge': '  Runs tests on Edge',
-  'ie': '  Runs tests on IE',
-  'chrome_canary': 'Runs tests on Chrome Canary',
-  'chrome_flags': 'Uses the given flags to launch Chrome',
-  'unit': '  Run only unit tests.',
-  'integration': '  Run only integration tests.',
-  'compiled':
-    '  Changes integration tests to use production JS ' +
-    'binaries for execution',
-  'grep': '  Runs tests that match the pattern',
-  'files': '  Runs tests for specific files',
-  'nohelp': '  Silence help messages that are printed prior to test run',
-  'a4a': '  Runs all A4A tests',
-  'coverage': '  Run tests in code coverage mode',
-  'headless': '  Run tests in a headless Chrome window',
-  'local-changes':
-    '  Run unit tests directly affected by the files ' +
-    'changed in the local branch',
-};
+test.description = 'Runs tests (DEPRECIATED)';
```

Comment:
DEPRECATED 🙃

Reply:
:see_no_evil: :see_no_evil: :see_no_evil: I meant to say that it lowered in value

Reply:
😃